File: Anul II/Medii de proiectare si programare/Laborator/lab07/Ubb.BikeContest.Networking/Request/GetRaceByNameRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ubb.BikeContest.Networking.Request
{
    [Serializable]
    public class GetRaceByNameRequest : IRequest
    {
        private readonly string raceName;

        public GetRaceByNameRequest(string raceName)
        {
            this.raceName = raceName;
        }

        public virtual string RaceName
        {
            get { return raceName; }
        }
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab10/networking/src/main/java/com/ubb/protocol/request/GetRaceByNameRequest.java
package com.ubb.protocol.request;

public class GetRaceByNameRequest implements Request {
    private final String raceName;

    public GetRaceByNameRequest(String raceName) {
        this.raceName = raceName;
    }

    public String getRaceName() {
        return raceName;
    }
}
